.github/workflows/lint-status-check.yml

```diff
@@ -0,0 +1,24 @@
+name: Lint
+
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  build:
+    name: Build
+    runs-on: macos-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set Node.js 10.13.0
+      uses: actions/setup-node@master
+      with:
+        node-version: 10.13.0
+
+    - name: Install dependencies
+      run: yarn
+
+    - name: Run lint check
+      run: yarn lint
```

Comment:
Instead of just a lint check, can we have a build check for master, which includes both tests and lint?